content/reference/sql/benchmarks/import.md

```diff
@@ -1,55 +1,46 @@
-
-- A CSV file with 100k rows in sorted key order
-- A CSV file with 100k rows in random key order
-- A CSV file with 1m rows in sorted key order
-- A CSV file with 1m rows in random key order
-- A CSV file with 10m rows in sorted key order
-- A CSV file with 10m rows in random key order
-
-For each CSV file we measured how long it took for the relevant file to get loaded in and the number of rows that were
-imported per second.
-
-## Benchmark Results
-
-Below are the results of the import benchmark. Like the [latency](latency.md) section, our goal is to get this to within 2-4x MySQL's performance. 
-
-```
-|      name       | program | version | from_time | from_rps | program | version | to_time  | to_rps  | rps_multiplier |
-|-----------------|---------|---------|-----------|----------|---------|---------|----------|---------|----------------|
-| 100k-sorted.csv | dolt    | 0.40.20 | 13.03s    | 7672.1   | mysql   | 8.0.22  | 1.32s    | 75571.4 | 9.9            |
-| 100k-random.csv | dolt    | 0.40.20 | 13.23s    | 7556.0   | mysql   | 8.0.22  | 1.38s    | 72207.3 | 9.6            |
-| 1m-sorted.csv   | dolt    | 0.40.20 | 2m29.29s  | 6698.2   | mysql   | 8.0.22  | 13.3s    | 75173.9 | 11.2           |
-| 1m-random.csv   | dolt    | 0.40.20 | 2m29.48s  | 6689.9   | mysql   | 8.0.22  | 12.76s   | 78361.3 | 11.7           |
-| 10m-sorted.csv  | dolt    | 0.40.20 | 27m42.44s | 6015.2   | mysql   | 8.0.22  | 2m14.22s | 74505.5 | 12.4           |
-| 10m-random.csv  | dolt    | 0.40.20 | 27m48.16s | 5994.6   | mysql   | 8.0.22  | 2m13.41s | 74956.6 | 12.5           |
-```
+Dolt supports three modes of import:
+
+1) [`LOAD DATA INFILE`](https://dev.mysql.com/doc/refman/8.0/en/load-data.html) SQL-server command.
+2) [`dolt table import`](../../cli.md#dolt-table-import) CLI command.
+3) `dolt sql < import.sql` batch script import.
+
+We recommend (1) > (2) > (3) for large import peformance. `dolt table import` is the most convenient and only slightly slower than `LOAD DATA INFILE`. Refer to the [import tutorial blog](https://www.dolthub.com/blog/2022-11-21-import-perf/) for a walkthrough of the different techniques.
+
+## Import comparison
+
+Each result row reports the runtime for a MySQL import of a certain table schema (`test_name` and `detail`), row count (`row_cnt`), and sorting (`sorting`). `sql_mult` and `cli_mult` distinguish the Dolt SQL (sql-server `LOAD DATA`) and CLI (`dolt table import`) latencys as a multiple of the MySQL latency for the same test conditions. All MySQL tests report the `LOAD DATA` latency.
+
+We are about 2x slower than MySQL for most import conditions. We are slightly slower importing blobs, reflecting how Dolt chunks blobs individual as prolly trees rather than a single byte array. Both Dolt and MySQL are less efficient importing sql scripts with standalone INSERT rows compared to batched insert script (we have a tool [here](https://github.com/dolthub/insert-batcher) to batch INSERT scripts).
```

Comment:
```suggestion
We are about 2x slower than MySQL for most import conditions. We are slightly slower importing blobs, reflecting how Dolt chunks blobs individually as prolly trees rather than a single byte array. Both Dolt and MySQL are less efficient importing sql scripts with standalone INSERT rows compared to batched insert script (we have a tool [here](https://github.com/dolthub/insert-batcher) to batch INSERT scripts).
```